src/App.js

```diff
@@ -1,7 +1,33 @@
 import "./App.css";
+import Header from "./components/Header";
+import { Route, Routes } from "react-router-dom";
+import Nav from "./components/Nav";
+import Articles from "./components/Articles";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Nav />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <p>
```

Comment:
These <p> tags around the components are unnecessary and should be removed

Reply:
hmmm true otherwise the homepage is nothing